test/unit/test_ArithLogicApi.cc

```diff
@@ -21,25 +23,31 @@ class ArithLogicApiTest: public ::testing::Test {
 };
 
 TEST_F(ArithLogicApiTest, test_LRA) {
+    SMTConfig config;
+    MainSolver solver(lraLogic, config, "test");
```

Comment:
We should not be using `MainSolver` in this file.

Reply:
Fixed